azurerm/internal/services/cosmos/cosmosdb_account_resource.go

```diff
@@ -277,6 +277,38 @@ func resourceCosmosDbAccount() *schema.Resource {
 				Default:  false,
 			},
 
+			"access_key_metadata_writes_enabled": {
+				Type:     schema.TypeBool,
+				Optional: true,
+				Default:  true,
+			},
+
+			"mongo_server_version": {
+				Type:     schema.TypeString,
+				Optional: true,
+				ForceNew: true,
+				Computed: true,
+				ValidateFunc: validation.StringInSlice([]string{
+					string(documentdb.ThreeFullStopTwo),
+					string(documentdb.ThreeFullStopSix),
+					string(documentdb.FourFullStopZero),
+				}, false),
+			},
+
+			"network_acl_bypass_for_azure_servers": {
```

Comment:
shouldn't this be
```suggestion
			"network_acl_bypass_for_azure_services": {
```
?